tests/optim/test_oss.py

```diff
@@ -472,24 +473,21 @@ def closure():
     dist.destroy_process_group()
 
 
-# TODO(blefaudeux) Fix for torch v1.8.0
-@pytest.mark.skipif(torch.__version__.split("+")[0].split(".") == ["1", "8", "0"], reason="disabled for torch 1.8.0")
+@skip_if_single_gpu
 def test_collect_shards():
-    world_size = 3
```

Comment:
3 was arbitrary, I think it was a copy paste from another test (multi groups) which required that